app/helpers/pagelets_helper.rb

```diff
@@ -0,0 +1,30 @@
+module PageletsHelper
+  def sorted_pagelets_at(mountpoint)
+    Pagelets::Manager.sorted_pagelets_at("#{controller_name}/#{action_name}", mountpoint)
+  end
+
+  def render_pagelets_at(mountpoint, opts)
+    result = ""
+    sorted_pagelets_at(mountpoint).each do |pagelet|
+      result += (render pagelet.partial, opts.merge!({ :pagelet => pagelet }))
+    end
+  end
+
+  def render_tab_content_at(mountpoint, opts)
+    result = ""
+    sorted_pagelets_at(mountpoint).each do |pagelet|
+      result += "<div id='#{pagelet.snake_name}' class='tab-pane'>"
+      result += (render pagelet.partial, opts.merge!({ :pagelet => pagelet }))
+      result +=  "</div>"
+    end
+    result.html_safe
+  end
+
+  def render_tab_header_at(mountpoint, opts)
+    result = ""
+    sorted_pagelets_at(mountpoint).each do |pagelet|
+      result += "<li><a href='##{pagelet.snake_name}' data-toggle='tab'>#{pagelet.name}</a></li>"
+    end
+    result.html_safe
+  end
+end
```

Comment:
Can you add one more method that simplifies the actual render line? So rendering in my testing case will be as easy as:

```ruby
  <% sorted_pagelets_at(:probe_contents).each do |pagelet| %>
    <div class="col-md-6">
      <%= render_pagelet(pagelet, :smart_proxy => @smart_proxy) %>
    </div>
  <% end %>
```